system_test/aest_sync_SUITE_data/epoch.yaml.mustache

```diff
@@ -21,6 +21,7 @@ websocket:
         port: {{services.int_ws.port}}
 
 keys:
+    # If the password is changed, reflect the change in aest_docker.erl
     password: "top secret"
```

Comment:
Can this be passed as root var in template? So at least it is only in aest_docker.erl

Reply:
We could do that.